Mathlib/MeasureTheory/Function/ConvergenceInMeasure.lean

```diff
@@ -58,6 +62,37 @@ theorem tendstoInMeasure_iff_norm [SeminormedAddCommGroup E] {l : Filter ι} {f
       ∀ ε, 0 < ε → Tendsto (fun i => μ { x | ε ≤ ‖f i x - g x‖ }) l (𝓝 0) := by
   simp_rw [TendstoInMeasure, dist_eq_norm]
 
+theorem TendstoInMeasure_of_FiniteMeasure [Dist E] {_ : MeasurableSpace α} {μ : Measure α}
+    [hfin: MeasureTheory.IsFiniteMeasure μ] {f : ι → α → E} {l : Filter ι} {g : α → E} :
+    TendstoInMeasure μ f l g ↔ ∀ ε, 0 < ε → Tendsto (ENNReal.toNNReal ∘ (fun i =>
+    (μ { x | ε ≤ dist (f i x) (g x) }))) l (𝓝 0) := by
```

Comment:
```suggestion
    TendstoInMeasure μ f l g ↔ ∀ ε, 0 < ε → Tendsto (fun i =>
    (μ { x | ε ≤ dist (f i x) (g x) }).toNNReal) l (𝓝 0) := by
```
This would be a more natural way of formulating

Reply:
Ok, changed.